cmd/manager/main.go

```diff
@@ -74,7 +83,7 @@ func main() {
 	}
 
 	mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
-		CertDir:    "/tmp/cert",
+		CertDir:    getEnv("KUDO_CERT_DIR", filepath.Join("/tmp", "cert")),
```

Comment:
`/tmp` isn't a valid path on Windows either. How about using [os.TempDir](https://golang.org/pkg/os/#TempDir)?

Reply:
Though, no strong opinion, we probably have other places as well that don't use Windows-compatible paths. Let's decide if we actually care about this and clean all paths up in a separate PR.

Reply:
fun fact: I just tested this on windows and it works. `filepath.Join("/tmp", "cert")` resolved to `C:\tmp\cert` which is a valid path.

Reply:
Interesting! Go magic is taking care of it ✨